files/common/lib/sysctl.d/50-override.conf

```diff
@@ -93,3 +93,10 @@ fs.mount-max = 300000
 # value of 8192, resulting in upgrade failures.
 #
 fs.inotify.max_user_watches = 32768
+
+#
+# We've seen cases where an NFSv4 delegation is revoked by the server,
+# which then results in client IO errors. We disable NFSv4 delegation
+# entirely here, to avoid the problem.
+#
+fs.leases-enable = 0
```

Comment:
Would this persist on a server reboot? I guess so but please confirm.

Reply:
Yes, that's the intention; I'll verify.

Reply:
Yes, it's persistent...
```
$ cat /proc/sys/fs/leases-enable
0

$ sudo reboot
Connection to ps-leases.dlpxdc.co closed by remote host.
Connection to ps-leases.dlpxdc.co closed.
⏎


> ssh delphix@ps-leases.dlpxdc.co
Warning: Permanently added 'ps-leases.dlpxdc.co' (ED25519) to the list of known hosts.

$ cat /proc/sys/fs/leases-enable
0

$ uptime
 20:19:38 up 3 min,  1 user,  load average: 2.57, 2.18, 0.96
 ```